charts/openstack-cluster/templates/node-group/openstack-machine-template.yaml

```diff
@@ -18,12 +18,16 @@ template:
     {{- if $nodeGroup.machineRootVolume.diskSize }}
     rootVolume: {{ toYaml $nodeGroup.machineRootVolume | nindent 6 }}
     {{- end }}
-    {{- if $ctx.Values.machineImageId }}
+    {{- if $ctx.Values.nodeGroupDefaults.machineImageId }}
+    imageUUID: {{ $ctx.Values.nodeGroupDefaults.machineImageId }}
```

Comment:
This should be `$nodeGroup.machineImageId` rather than `$ctx.Values.nodeGroupDefaults.machineImageId` - see above.

Reply:
I assume that should also apply to `$ctx.Values.nodeGroupDefaults.machineImage` right below that?

Reply:
I changed the `$nodeGroup.machineImageId` and also `$nodeGroup.machineImage` as well.